qutip/entropy.py

```diff
@@ -338,39 +337,3 @@ def participation_ratio(rho):
         return 1.0
     else:
         return 1.0 / (rho ** 2).tr()
-
-
-def entangling_power(U):
-    """
-    Calculate the entangling power of a two-qubit gate U, which
-    is zero of nonentangling gates and 1 and 2/9 for maximally
-    entangling gates.
-
```

Comment:
What do we do with entangling power now? Do we still delete it or move it to `qutip.gate`

Reply:
It is a bit restricted but might be useful for those who study quantum information.

Reply:
We should keep `entangling_power` in `entropy`. It mentions the word "gate" in its docstring but really it calculate the entangling power of a two qubit operator, which is meaningful outside of the realm of QIP.

Reply:
I also don't entirely understand what it means, probably it should be from 0 to 2/9 from what I see below. But strangely for `swap` it is also 0...
```python
>>> qutip.entangling_power(qutip.cnot())
<stdin>:1: DeprecationWarning: Importing functions/classes of the qip submodule directly from the namespace qutip is deprecated. Please import them from the submodule instead, e.g.
from qutip.qip.operations import cnot
from qutip.qip.circuit import QubitCircuit

0.22222222222222227
>>> qutip.entangling_power(qutip.qeye([2,2]))
0.0
>>> qutip.entangling_power(qutip.swap())
0.0
>>> qutip.entangling_power(qutip.iswap())
0.22222222222222227
>>> qutip.entangling_power(qutip.sqrtiswap())
0.16666666666666674
>>> qutip.entangling_power(qutip.sqrtswap())
0.16666666666666674
```

Reply:
The swap gate does not produce any entanglement since it swaps the two qubit completely. One needs to do a fractional swap to such as sqrtswap to get entanglement.

The iswap gate has maximal entangling power as a result of the multiplication by i of one basis element from each qubit (i.e. the "i" part). The swap part doesn't contribute to the entangling power.

Reply:
The documentation is wrong but this is not the first QuTiP function for which this is the case. ;)